### ISSUE TITLE
don't crash if fuzzy not found

### ISSUE THREAD

**Issue by GopherJ** [ajeetdsouza/zoxide#129]
currently zoxide crashes while not founding target path in db.

```
Error: no match found

Stack backtrace:
   0: <zoxide::cmd::App as zoxide::cmd::Cmd>::run
   1: zoxide::main
   2: std::sys_common::backtrace::__rust_begin_short_backtrace
   3: main
   4: __libc_start_main
   5: _start
```

is it possible to just ignore jumping instead of crashing?

like https://github.com/xen0n/autojump-rs

**Comment by ajeetdsouza** [comment]
That's not actually a crash - `Error: no match found` is printed to make it obvious that the directory hasn't been changed. The backtrace, OTOH, shouldn't be there. Are you setting any Rust-specific env vars? What is `zoxide --version`?

**Comment by GopherJ** [comment]
@ajeetdsouza maybe it's caused by my:

```
export RUST_BACKTRACE=1
```

**Comment by ajeetdsouza** [comment]
You're probably using nightly to compile, too. This is from the `anyhow` docs:

> If using the nightly channel, a backtrace is captured and printed with the error if the underlying error type does not already provide its own. In order to see backtraces, they must be enabled through the environment variables described in `std::backtrace`.

I've fixed this in #130.

**Comment by GopherJ** [comment]
@ajeetdsouza thanks!
